.travis.yml

```diff
@@ -0,0 +1,22 @@
+sudo: false
+language: python
+env:
+  global:
+    - CODECOV_TOKEN=003330f1-4898-4761-b1ea-555084097df0
```

Comment:
I believe, since this is an open source repo, you don't need to include the token. Per their example project: https://github.com/codecov/example-python/blob/master/.travis.yml